datasets/geonet.yaml

```diff
@@ -0,0 +1,38 @@
+Name: GeoNet Aotearoa New Zealand Data
+Description: GeoNet provides geological hazard information for Aotearoa New Zealand. This dataset contains data and products recorded by the GeoNet sensor network. The dataset currently include GNSS data and additional datasets will be added in the near future.
+  GNSS (Global Navigation Satellite System) data include raw data in proprietary and Receiver Independent Exchange Format (RINEX) and local tie-in survey conducted during equipment changes '(https://www.geonet.org.nz/data/types/geodetic)'.
+Documentation: https://github.com/GeoNet/data-tutorials/tree/main/AWS_Open_Data
+Contact: For questions regarding data content or quality, please visit '[the GeoNet site](https://www.geonet.org.nz/)]', contact info@geonet.org.nz or use GitHub issue on [GeoNet/help](https://github.com/GeoNet/help).
+ManagedBy: "[GeoNet](https://www.geonet.org.nz/)"
+UpdateFrequency: Daily for majority of datasets.
+Tags:
+  - aws-pds
+  - earth science
+  - geophysics
+  - geological hazard
+  - natural hazard monitoring
+  - tectonics
+  - geodesy

```

Comment:
"earth science, geological hazard, natural hazard monitoring, geodesy, tectonics" not supported tags.  Please have a look at https://github.com/awslabs/open-data-registry/blob/main/tags.yaml and update accordingly.  Thanks!

Reply:
thanks a lot @jeffejefe ! will do

Reply:
thanks @elidana the supported tags aren't the most obvious but its a common reason for the error.

Reply:
tags now updated, thanks @jeffejefe . 

If possible, would be ok to raise a (separate) pull request to add a few more? I see that there are some agency specific tags that have been used.